css/sass/_mega-menu.scss

```diff
@@ -65,7 +65,7 @@
 }
 
 .minus {
-  background: url(../img/minus.png) 5px 12px no-repeat !important;
```

Comment:
@punalsc I think we should address this with specificity rather than an `!important` rule. The [development guide](https://github.com/nationalarchives/development-guide-and-peer-reviews/blob/master/development-guide.md) says that we should avoid using `!important`

I also wonder if `expanded` might be a more descriptive name than `minus`.